### ISSUE TITLE
TPV  ubiquitin-dependent protein catabolic process

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12370]
GO:0071629	cytoplasm-associated proteasomal ubiquitin-dependent protein catabolic process
GO:0071630	nucleus-associated proteasomal ubiquitin-dependent protein catabolic process

are both descendants of
GO:0071218 cellular response to misfolded protein

should not be

**Comment by mcourtot** [comment]
Could expand on why not? Sorry, the issue doesn't seem immediately obvious to me.

**Comment by ValWood** [comment]
ubiquitin dependent catabolism  isn't always a cellular response to misfolded protein. For example lots of proteins are  catabolised by the proteasome in a cell cycle dependent manner. These aren't misfolded.

**Comment by mcourtot** [comment]
Checking the definitions:
- GO:0071629 cytoplasm-associated proteasomal ubiquitin-dependent protein catabolic process is defined as "The chemical reactions and pathways resulting in the breakdown of misfolded proteins in the cytoplasm, which are targeted to cytoplasmic proteasomes for degradation."
- GO:0071630 nucleus-associated proteasomal ubiquitin-dependent protein catabolic process is defined as "The chemical reactions and pathways resulting in the breakdown of misfolded proteins via a mechanism in which the proteins are transported to the nucleus for ubiquitination, and then targeted to proteasomes for degradation."

So it seems in this case these terms are *only* for response to misfolded proteins. 

The parent term GO:0043161, proteasome-mediated ubiquitin-dependent protein catabolic process, defined as "The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds, initiated by the covalent attachment of ubiquitin, and mediated by the proteasome." doesn't have the same relationship, so maybe this would be better suited for the general case?

**Comment by ValWood** [comment]
Woah that's misleading. These processes  definitely occur on non-misfolded proteins... (in a ubiquitin dependent manner).

**Comment by ValWood** [comment]
Oh its not a big problem. Only a small number of EXP annotations (SGD and PomBase).
Can we make it clearer from the term name that this refers to misfolded proteins - I can reannotate ours where necessary.

**Comment by mcourtot** [comment]
I agree that it would be useful to make it clearer - do you have a suggestion for new label? Something like cytoplasm-associated proteasomal ubiquitin-dependent protein catabolic process in response to misfolded protein maybe?

**Comment by ValWood** [comment]
Some of these labels are unnecessarily long :)

misfolded cytoplasmic protein catabolic process via the proteasome

might be enough.
I had a quick glance at the paper, its referring to  cytoplasmic proteins, but i don't know if the proteasomal degradation necessarily occurs in the cytoplasm.

Probably don't need to mention "ubiquitin-dependent" in the term name, only the def as all proteasomal  is ubiquitin dependent?

**Comment by mcourtot** [comment]
pinging @dosumis as he did work on refactoring long names.

The hierarchy currently looks like:
- ubiquitin-dependent protein catabolic process
---- proteasome-mdeiated ubiquitin-dependent protein catabolic process
---------- cytoplasm-associated proteasomal ubiquitin-dependent protein catabolic process

So despite the long name, cytoplasm-associated proteasomal ubiquitin-dependent protein catabolic process in response to misfolded protein would be consistent.

**Comment by mcourtot** [comment]
pinging @dosumis for input on the label.
@dosumis: current proposal is to rename
GO:0071629 cytoplasm-associated proteasomal ubiquitin-dependent protein catabolic process
GO:0071630 nucleus-associated proteasomal ubiquitin-dependent protein catabolic process
to
GO:0071629 cytoplasm-associated proteasomal ubiquitin-dependent protein catabolic process in response to misfolded protein
GO:0071630 nucleus-associated proteasomal ubiquitin-dependent protein catabolic process in response to misfolded protein

Those labels are quite long, but consistent with the hierarchy. 
@ValWood suggested instead to use misfolded cytoplasmic protein catabolic process via the proteasome and misfolded nucelic protein catabolic process via the proteasome

Could you indicate which one works best given the label refactoring you did?

**Comment by dosumis** [comment]
The original terms could be made much clearer like this:

cytoplasm-associated proteasomal ubiquitin-dependent protein catabolic process
>
ubiquitin-dependent protein catabolism by cytoplasm-associated proteasome

If we really need terms for Ubiq dependent catabolism of misfolded proteins by the proteasome:

ubiquitin-dependent of catabolism of misfolded proteins by cytoplasm-associated proteasome

**Comment by ValWood** [comment]
Some very clunky terms around here, and a number probably unnecessary. LEGO will help around this area but it would help in the short term if the term names reflected the definitions more closely, and  the definitions specifically refer to misfolded proteins.

**Comment by mcourtot** [comment]
Consensus is:

GO:0071629 ubiquitin-dependent of catabolism of misfolded proteins by cytoplasm-associated proteasome
and
GO:0071630 ubiquitin-dependent of catabolism of misfolded proteins by nucleus-associated proteasome

Let me know if you object; otherwise i'll implement tomorrow.

Thanks for the feedback!
